docs/Configuration/fleet-server-configuration.md

```diff
@@ -2213,9 +2213,9 @@ Name of the S3 bucket for storing software and file carves.
     bucket: some-bucket
   ```
 
-##### s3_prefix
+##### s3_software_installers_prefix
 
-Prefix to prepend to software and file carves.
+Prefix to prepend to software.
 
 All carve objects will also be prefixed by date and hour (UTC), making the resulting keys look like: `<prefix><year>/<month>/<day>/<hour>/<carve-name>`.
 
```

Comment:
I can't comment on that line because it is not part of the diff, but for all the config options in this PR, the `Environment variable` line must be changed to represent the new name (e.g. for this current section it would be `FLEET_S3_SOFTWARE_INSTALLERS_PREFIX`, not `FLEET_S3_PREFIX`).

Reply:
Same thing for the `Config file format` YAML example, it should be e.g. for this config:

```
s3:
    software_installers_prefix: prefix-here/
```

Reply:
Thanks! I was moving too fast..